src/SFA.DAS.EAS.Web/Views/EmployerAccountTransactions/TransactionsView.cshtml

```diff
@@ -0,0 +1,240 @@
+aggregationLine.TransactionType == TransactionItemType.TopUp)
+                                {
+                                    <a title="View details for the transaction on @date" href="@Url.Action("LevyDeclarationDetail", "EmployerAccountTransactions", new {HashedAccountId = Model.Data.Model.Data.HashedAccountId, fromDate, toDate})">
+                                        Details <span class="vh"> for the transaction on @date</span>
+                                    </a>
+                                }
+                                else if (aggregationLine.TransactionType == TransactionItemType.Payment)
+                                {
+                                    var paymentTransaction = aggregationLine as PaymentTransactionLine;
+
+                                    <a title="View details for the transaction on @date" href="@Url.Action("ProviderPaymentSummary", "EmployerAccountTransactions", new {HashedAccountId = Model.Data.Model.Data.HashedAccountId, ukPrn = paymentTransaction?.UkPrn, fromDate = fromDate, toDate = toDate })">
+                                        Details <span class="vh"> for the transaction on @date</span>
+
+                                    </a>
+                                }
+                            </td>
+                        </tr>
+                    }
+                </tbody>
+            </table>
+        }
+
+        <ul class="previous-next-navigation">
+            @if (!Model.Data.IsFirstMonthOfLevy && Model.Data.AccountHasPreviousTransactions)
+            {
+                <li class="previous">
+                    <a href="@Url.Action("TransactionsView", new {year = @previousMonthDate.Year, month=@previousMonthDate.Month})">
+                        Previous <span class="hide-mobile">month</span>
+                        <span class="detail">@previousMonthDate.ToGdsFormatWithoutDay()</span>
+                    </a>
+                </li>
+            }
+            @if (!Model.Data.IsLatestMonth)
+            {
+                <li class="next">
+                    <a href="@Url.Action("TransactionsView", new {year = @nextMonthDate.Year, month=@nextMonthDate.Month})">
+                        Next month
+                        <span class="detail">@nextMonthDate.ToGdsFormatWithoutDay()</span>
+
+                    </a>
+                </li>
+            }
+        </ul>
+
+        <details class="feedback">
+            <summary><span class="summary">Help us improve this section</span></summary>
+            <div class="survey">
+                <script id="ss-embed-396029">
+                    (function (d, w) {
+                        var s, ss; ss = d.createElement('script');
+                        ss.type = 'text/javascript';
+                        ss.async = true;
+                        ss.src = ('https:' == d.location.protocol ? 'https://' : 'http://') + 'www.smartsurvey.co.uk/s/r/embed.aspx?i=350750&c=396029';
```

Comment:
can we add this url to the config?

Reply:
This is the only usage, and I have no requirements over it, it was copy-paste existing code, so I think if this is required it is part of a bigger tech debt piece of work.